#! /bin/sh -e
## 01_conffiles.dpatch by Joerg Jaspert <joerg@debian.org>
## (original made by some of the former maintainers)
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Comments USER and ACCESS entries in rscsi file.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad cdrtools-2.01.01~/rscsi/rscsi.dfl cdrtools-2.01.01/rscsi/rscsi.dfl
--- cdrtools-2.01.01~/rscsi/rscsi.dfl	2005-08-09 15:19:20.000000000 +0200
+++ cdrtools-2.01.01/rscsi/rscsi.dfl	2006-03-20 21:02:43.116442448 +0100
@@ -31,7 +31,7 @@
 #
 # USER= entries are checked with a pattern matcher. USER=* matches all users.
 #
-USER=rscsi
+#USER=rscsi
 #USER=joerg
 
 #
